exoseq.nf

```diff
@@ -47,15 +59,15 @@ Other options:
 --project                      Uppnex project to user for SLURM executor
 
 For more detailed information regarding the paramaters and usage refer to package
-documentation at https://github.com/SciLifeLab/NGI-EXOseq
+documentation at https:// github.com/SciLifeLab/NGI-ExoSeq
 """
 
-//Variables and defaults
+// Variables and defaults
 params.help = false
 params.reads = false
 params.genome = false
 params.clusterOptions = false
-params.project = 'ngi2016003'
+params.project = false
 params.outdir = './results'
```

Comment:
No, all `params` should be explicitly defined in the main pipeline script. Config options can overwrite these defaults, but they need to be defined here.